tests/language-service/languageservicerunner.ts

```diff
@@ -46,16 +47,34 @@ describe("language service", () => {
 })
 
 function getTestCases() {
-    const filenames: string[] = [];
+    let filenames: string[] = [];
     for (const file of fs.readdirSync(casesDir)) {
+        // ignore hidden files
         if (file[0] == ".") {
             continue;
         }
 
+        // ignore files that start with TODO_; these represent future work
+        if (file.indexOf("TODO") >= 0) {
+            continue;
```

Comment:
Emit a warning?

Reply:
Hmm, I don't think a warning is a appropriate here. This is a great way to add a test case that we can reference in an Issue. The priority of this should be decided in the Issue and might not justify a warning every time. We know there will be lots of features we don't get to for a while. This is also a way to document which language service features are missing.

Reply:
I disagree this becomes a secret test case that no one cares about. We should emit a message in the console stating we skipping a test. Test do get lost if they are not run.

Reply:
I'm okay with the tests becoming "lost". They can be found when they become relevant, e.g. if we decide a feature is important for users or an issue gets raised in priority. Someone exploring this part of the code, e.g. if I go on another intellisense tear in the future, can discover these easily enough.

Warnings that are always present get lost just as easily.
Everytime anyone runs pxt serve we output:
"error TS1084: Invalid 'reference' directive syntax."
no one cares. I believe this has been there for >1 year.

Reply:
I've noticied that my future self forgets these kinds of subtle semantics (TODO means don't run). Adding a simple console.log today might pay off saving a few hours in the future.

> Warnings that are always present get lost just as easily.

They come back on every build.

Reply:
Worth noting that I documented the behavior of TODO_ in the .md file for this test suite.

I don't feel strongly about this anymore, so I'm fine adding a warning during my next PR if you still feel strongly.